server.ts

```diff
@@ -0,0 +1,66 @@
+import 'zone.js/dist/zone-node';
+
+import { ngExpressEngine } from '@nguniversal/express-engine';
+import * as express from 'express';
+import { join } from 'path';
+
+import { AppServerModule } from './src/main.server';
+import { APP_BASE_HREF } from '@angular/common';
+import { existsSync } from 'fs';
+
+// The Express app is exported so that it can be used by serverless Functions.
```

Comment:
Let's remove all the comments here as they don't add much value. If we want to keep some then we need to update the style of the comments. Inline comments should be lowercase and not end on punctuation. For multi-line comments the convention is to use `/** */` instead of multiple `//`.